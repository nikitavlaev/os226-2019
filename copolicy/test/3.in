deadline
rtapp 1 -1 2
rtapp 2 2 2
rtapp 1 -1 2
