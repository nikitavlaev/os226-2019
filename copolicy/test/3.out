rtapp id 1 cnt 1
rtapp id 1 cnt 0
rtapp id 0 cnt 1
rtapp id 2 cnt 1
rtapp id 0 cnt 0
rtapp id 2 cnt 0
